src/canonicalize.d.ts

```diff
@@ -0,0 +1 @@
+declare module 'canonicalize';
```

Comment:
see above

Reply:
This is required in order for a TS module to reference a JS module. Its basically adding missing type information so that the compiler knows about the untyped module and its contents. In this case the contents are unimportant so only the fact that the module exists is  present